lib/shopify_cli/theme/theme.rb

```diff
@@ -9,12 +10,11 @@ module ShopifyCLI
   module Theme
     class InvalidThemeRole < StandardError; end
 
-    class Theme
+    class Theme < Root
       attr_reader :root, :id
 
       def initialize(ctx, root: nil, id: nil, name: nil, role: nil)
-        @ctx = ctx
-        @root = Pathname.new(root) if root
+        super(ctx, root: root)
```

Comment:
should we do the same thing here that we did with app extension to use delegators with Root? if so i can open an issue to do this